docs/tutorials/hello-world.md

```diff
@@ -15,161 +15,192 @@ First we need to make sure that the [sCrypt CLI tool](https://github.com/sCrypt-
 We can run the following command to create a new sCrypt project:
 
 ```sh
-scrypt project my-project
+npx scrypt-cli project hello-world
```

Comment:
Let's always use globally installed CLI in the doc, as in CRA or snarkyjs-cli. npx is better needed in prod.